test/helper.js

```diff
@@ -1,32 +1,37 @@
 'use strict'
-
 const { Grape } = require('./../')
 
 exports.createGrapes = createGrapes
 function createGrapes (n, onstart) {
   const grapes = []
   let missing = n
 
+  const bootstrap = new Grape({
```

Comment:
bootstrap server behaviour in hyperswarm dht vs bittorrent dht seems to differ, needs more investigation but for now just creating a single boootstrap node

Reply:
does it affect https://github.com/bitfinexcom/bfx-svc-test-helper/blob/master/grapes.js ?

Reply:
possibly - I need look more into the behaviours, but if it's ok I'll circle back to this later and make sure it works with  https://github.com/bitfinexcom/bfx-svc-test-helper/blob/master/grapes.js